ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended~acs-commons.cfg.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
cfg.json is only supported since AEMaaCS, we still have to support AEM 6.3 Please use .config!